### PR TITLE
added info on the DevOps concepts

### PR DESCRIPTION
**Pull request by rishabkumar7**

### CODE REVIEW

**Review by madebygps — APPROVED** [review]
lgtm

File: docs/phase4/README.md [hunk 1]
```diff
@@ -21,15 +21,35 @@ You should've completed the [Cloud Resume Challenge](https://cloudresumechalleng
...
+
+### Containerization
+
+Containerization is the packaging of software code with just the operating system (OS) libraries and dependencies required to run the code to create a single lightweight executable—called a container, that runs consistently on any infrastructure. More portable and resource-efficient than virtual machines (VMs). They have become the de facto compute units of modern cloud-native applications.
+
+### Monitoring
+
+Monitoring allows you to keep track of the health and performance of software and systems. It involves collecting useful data about system's performance and behavior, such as CPU usage, network traffic, memory usage and response times. Monitoring tools also provide real-time alerts when something goes wrong, allowing you to quickly identify and respond to resolve issues before they become major problems.
+
+Feel free to spend as much time as you'd like, people have asked for a timeline and breakdown so I've added it. Every DevOps practice matters, but these are the most used on an entry/junior level. You should already be familiar with Version Control from Phase 2.
```
[the remark this answers is not in the document]
Comment (shivam-sharma7) [reply docs/phase4/README.md:52]: It would be better to remove extra line.

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

**Comment by madebygps** [comment]
hi @shivam-sharma7 those lines are for markdown formatting.
@rishabkumar7 please merge your changes only today.

**Comment by shivam-sharma7** [comment]
okay @madebygps

**Pull request merged**